src/Monai.Deploy.WorkflowManager.Storage/Services/DicomService.cs

```diff
@@ -0,0 +1,30 @@
+﻿// SPDX-FileCopyrightText: © 2021-2022 MONAI Consortium
+// SPDX-License-Identifier: Apache License 2.0
+
+using Ardalis.GuardClauses;
+using Monai.Deploy.Storage;
+
+namespace Monai.Deploy.WorkflowManager.Storage.Services
+{
+    public class DicomService : IDicomService
+    {
+        private IStorageService StorageService { get; set; }
```

Comment:
nit: any reason to use a property instead of field?

Reply:
No reason, changed to be a field